File: Java/java practice/MultiplicationTable.java
import java.util.Scanner;

public class MultiplicationTable {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter the table of number:");
        int num = sc.nextInt();
        int Table;
        System.out.println("Table of a number :"+num);
        for(int i=1;i<=10;i++){
            Table = num*i;
            System.out.println(Table);

        }

        
    }
    
}

File: c tutorial/only for practice c/multiplication_table.c
#include<stdio.h>

int main()
{
    int num;
    //take the number as an input from user
    printf("enter the value of whose multiplication table is to be printed:");
    scanf("%d",&num);
    printf("the multiplication table of %d is\n",num);
    for(int i=1;i<=10;i++)
    {
        printf(" %d X %d = %d\n",num,i,i*num);
    }
    return 0;
}